src/Search.js

```diff
@@ -17,7 +18,7 @@ const Search = () => {
                 className="form-control"
```

Comment:
Consider adding `required` to this element to prevent accidentally clicking the search button without putting something in it first